Makefile

```diff
@@ -99,7 +99,9 @@ docs-revise:
 
 check-whitespace:
 ifneq ($(NO_GIT), 1)
```

Comment:
@staticfloat @StefanKarpinski Does the whitespace check still require Git? If not, we don't need this conditional, right?

Reply:
Looks to me like it does: https://github.com/JuliaLang/julia/blob/a761cc0b82f88838eb31b29e7387cea35a0cab30/contrib/check-whitespace.jl#L23